### ISSUE TITLE
Datepicker not showing

### ISSUE THREAD

**Issue by four7** [YourITGroup/OpenOrClosed#50]
After installing the package I realized I couldnt see the datepick in the settings section when adding the datatype. I figured that maybe I can only see it in the content page but there I couldn't see either datepicker or any other inputs.

This is using Umbraco 11.x

**Comment by robertjf** [comment]
firstly, what specific version of Umbraco and the package are you using?  I've been using OpenOrClosed with Umbraco 11 successfully without any issues at all, so it would be helpful to know specifics.

Secondly, open up Dev Tools on your browser - are there any errors in the console?  Please add them to this ticket...

**Comment by four7** [comment]
We are on Umbraco 11.3.0 and no, no devtools errors, this is why I couldn't figure out myself what the issue was..since it showed no errors.

**Comment by robertjf** [comment]
hm.  I'll do some additional testing.  What browser are you using?

**Comment by four7** [comment]
Chrome Dev :)

**Comment by chrno1209** [comment]
I couldn't see the editor too, I am using Umbraco v10.5

**Comment by robertjf** [comment]
Hi @four7 I'm unable to replicate this - are you able to run up a sample website and if you can replicate it, zip it up and share it with me?

**Comment by robertjf** [comment]
@yourbabyscan is this a fresh install?  Does it work locally vs. when you deployed it?  Are there any errors in the Umbraco log?

**Comment by yourbabyscan** [comment]
It's a fresh install of Umbraco, and then OpenOrClosed added to it, yeah... not tried locally... I'll give that a go now.

**Comment by yourbabyscan** [comment]
Annoyingly, it works locally. I thought it might have been because I'm working on the site and it doesn't have a valid SSL cert on the deployed site yet... but after getting one and deploying it, still not appearing on the deployed version. 

Will get the remote directory and do a compare locally to see if I can spot the difference, I'm clearly missing something.

**Comment by yourbabyscan** [comment]
There were no file differences between the remote host and local (other than partial views created in the CMS on the remote deployment - expected). I tested bringing them local to make sure one of them wasn't causing the issues, but still worked fine locally. 

Looks like it could be the same issue with smidge as reported here:
https://our.umbraco.com/forum/umbraco-9/107740-umbraco-9-cant-register-controller-for-a-custom-dashboard-that-works-locally-but-fails-when-deploy

I realised I don't have an appsettings.production.json, and the appsettings.json doesn't include:

"Umbraco": {
    "CMS": {
      "RuntimeMinification": {
        "useInMemoryCache": true,
        "cacheBuster": "Timestamp"
      }
    }
  }

Just deploying and testing now, will update with my results.
